tests/unit/test_charm.py

```diff
@@ -173,7 +173,7 @@ def test_healthy_fails_if_snap_not_active(harness: Harness[KafkaCharm], zk_data,
         assert not harness.charm.broker.healthy
         assert patched_snap_active.call_count == 1
         if SUBSTRATE == "vm":
-            assert harness.charm.unit.status == Status.SNAP_NOT_RUNNING.value.status
+            assert harness.charm.unit.status == Status.BROKER_NOT_RUNNING.value.status
         elif SUBSTRATE == "k8s":
             assert harness.charm.unit.status == Status.SERVICE_NOT_RUNNING.value.status
```

Comment:
**question**: I noticed that this status does not exist. Am I missing something?

Reply:
No, I was talking about `SERVICE_NOT_RUNNING`, sorry I was not clear. But I got my answer, it is specific to K8s. Is there any chance we can have 1:1 parity in the two code bases to avoid linter warnings?